src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { useNotesContext } from "../../utils/contexts/NotesContext";
+import ColorOptions from "../ColorOptions";
+import "./Modal.styles.css";
+function Modal(props) {
+  const { note, closeEditing } = props;
+  const { notes, saveNotes } = useNotesContext();
+  const [values, setValues] = useState(note);
+  const [openMenu, setOpenMenu] = useState(false);
+
+
+  const { title, text, color } = values;
+
+  const editNote = () =>{
+    const newList = [...notes];
+    const edited = newList.find((f) => f.id === note.id);
+    edited.title = values.title;
+    edited.text = values.text;
+    edited.color = values.color;
+    saveNotes(newList);
+    closeEditing();
+  };
+
+  const changeColor = (color) => {
+    if (color === values.color) setValues({ ...values, color: "default" });
+    else setValues({ ...values, color: color });
```

Comment:
if the condition is true the values will be updated, if the condition is false the values will be updated as well, what's changing here is the color of the note, the readability could be improved with something like:

```jsx
const newColor = color === values.color
  ? 'default'
  : color;

setValues({ ...values, color: newColor });
```